frontend/src/pages/org/archived-item-qa/archived-item-qa.ts

```diff
@@ -535,6 +552,62 @@ export class ArchivedItemQA extends BtrixElement {
               <sl-icon name="replaywebpage" library="app"></sl-icon>
               ${msg("Replay")}
             </btrix-navigation-button>
+            <div class="ml-auto flex items-center gap-3">
+              ${when(
+                !this.analyzed,
+                () =>
+                  html`<btrix-popover
+                    content=${msg(
+                      "Screenshot, text, and resource quality metrics are only available for analyzed crawls. Run analysis to view and compare all QA metrics.",
+                    )}
+                  >
+                    <div
+                      class="flex items-center gap-1.5 text-xs text-neutral-500"
+                    >
+                      <sl-icon class="text-sm" name="info-circle"></sl-icon>
+                      ${msg("Limited view")}
```

Comment:
Open to suggestions on how to label and word this (cc @DaleLore )

<img width="386" height="125" alt="Screenshot 2025-07-14 at 12 05 15 PM" src="https://github.com/user-attachments/assets/8fea1653-f762-45b2-9ee4-ca2cc17c2011" />

Reply:
Might include "comparison" in a couple places? i.e. 

> Screenshot comparison, text comparison, and resource quality metrics are only available for analyzed crawls. Run analysis to view and compare all QA metrics.